client/ayon_houdini/plugins/create/create_FBX.py

```diff
@@ -122,7 +137,7 @@ def get_dynamic_data(
         The default prodcut name templates for Unreal include {asset} and thus
         we should pass that along as dynamic data.
         """
-        dynamic_data = super(CreateStaticMesh, self).get_dynamic_data(
+        dynamic_data = super(CreateFBX, self).get_dynamic_data(
```

Comment:
```suggestion
        dynamic_data = super().get_dynamic_data(
```
Should be fine in Py3.